src/main/java/firebase/sdk/dashboard/dao/PlatformReleaseDaoDatastore.java

```diff
@@ -0,0 +1,53 @@
+package firebase.sdk.dashboard.dao;
```

Comment:
I guess these changes are not supposed to be in this PR. Can you please move them to a separate branch/PR.